### DISCUSSION TITLE
Level editor - Entity property manipulation?

### DISCUSSION THREAD

**Discussion by SenorRagequit** [exorstudios/riftbreaker-tools#16] [Wishlist] [answered]
(Disclaimer: Maybe it's there and I haven't found it yet)

Normally, level editors feature at least a basic set of entity property manipulation, which seem to be missing here completely?
Here you can place an entity, transform it, and that's it.

In most other games you select 1 entity or N of the same type and get a small entity gui where you can change things like:
- HP
- Team (Player, enemy, neutral or passive)
- Size (We got that!)
- Speed
- Damage
- xyz

So I wonder IF that would ever come to this editor, or will it stay like this, and we have to edit the files generated?

**Comment by lukaasm** [comment, marked as answer]
It was/is there, but was reduced to only one component with 'Rename entity' entry in the context menu.

It really didn't work with our constant iterational development, because when you have attached/overridden a component on an entity placed on scene it would later ignore all changes done to that component done in the EntityBlueprint file.

For ex, you place a tree on tile and from editor add HealthComponent to it with 100hp then later designer decides that this tree blueprint should have HealthComponent with 250hp by default and places it in EntityBlueprint. No one remembers that somebody manually changed its HP on one of the tiles, and it will be forever locked to 100HP ignoring all changes done to this component in the base blueprint :)

For modding maybe it would be fine since you are crafting custom experience and you may want to have entity changes limited to scope of the current mission only.
